src/UI/Components/TreeTable/TreeRow/InstanceCellButton.tsx

```diff
@@ -5,7 +5,7 @@ import { useGetInstance } from "@/Data/Queries";
 interface Props {
   id: string;
   serviceName: string;
-  onClick: (cellValue: string, serviceName?: string | undefined) => void;
+  onClick: (cellValue: string, serviceName?: string | undefined, instanceId?: string) => void;
 }
 
 export const InstanceCellButton: React.FC<Props> = ({ id, serviceName, onClick }) => {
```

Comment:
docstrings ;)

Reply:
I'll add them, but keep the scope in mind of a PR, because otherwise, we can just go add doctstrings over the entire codebase in one go :P